sources/packages/backend/apps/api/src/route-controllers/models/pagination.dto.ts

```diff
@@ -118,6 +118,17 @@ export class OfferingsPaginationOptionsAPIInDTO extends PaginationOptionsAPIInDT
   sortField?: string;
 }
 
+export class CASInvoiceBatchesPaginationOptionsAPIInDTO extends PaginationOptionsAPIInDTO {
+  @IsOptional()
+  @IsIn(["batchDate"])
+  sortField?: string;
+  @IsOptional()
+  @IsArray()
+  @Transform(({ value }) => value.split(","))
```

Comment:
I couldn't follow the usage of this decorator here. I see it is used in programs search as well.

Reply:
The query string will be provided as a regular comma-separated string in the URL, for instance, myQueryarameter=item1,item2,item3. This converts the comma-separated string to a nice array to be consumed by the controller. Besides converting it, the decorators will ensure the array items are also expected enum values.

Reply:
Sounds good.